docs/conceptual/omnitrace-feature-set.rst

```diff
@@ -128,8 +128,8 @@ broad picture.
 
 .. note::
 
-   AMD is working on a separate tool that provides insight into the execution of individual kernels
-   on the GPU.
+   For insight into the execution of individual kernels on the GPU, 
+   use `Omniperf <https://rocm.github.io/omniperf/index.html>`_.
```

Comment:
Yes, I wasn't quite sure what to link to right now given that the new Omniperf documentation isn't live yet. Maybe the GitHub link is better because it's going to remain active. We can change it to the docs whenever we have a chance at some later point.

Reply:
I changed it to the GitHub link for now.